src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Truck started working!");

```

Comment:
The message "Truck started working!" could be more informative by indicating the specific type of machine that is working. Consider including the class name in the message to make it clear.